lua/damagelogs/client/tabs/rdm_manager.lua

```diff
@@ -1058,8 +1072,10 @@ function PANEL:SetPlayer(reported, ply, steamid, report)
     self.Button.DoClick = function(panel)
         if IsValid(ply) then
             if ulx then
-                RunConsoleCommand("ulx", mode == 1 and "aslayid" or "ajailid", ply:SteamID(), tostring(self.NumSlays), self.CurrentReason)
-            else
+                RunConsoleCommand("ulx", mode == 1 and "aslay" or "ajail", ply:Nick(), tostring(self.NumSlays), self.CurrentReason)
```

Comment:
Can this go back to `aslayid`/`ajailid`?

I remember this command having problems in the past with players that had weird characters in their names.

Reply:
done